assets/src/components/detours/diversionPanel.tsx

```diff
@@ -1,11 +1,12 @@
-import React, { ReactNode } from "react"
+import React from "react"
 import { RoutePill } from "../routePill"
 import { DetourShape } from "../../models/detour"
 import { ListGroup } from "react-bootstrap"
+import { Stop } from "../../schedule"
 
 export interface DiversionPanelProps {
   directions?: DetourShape["directions"]
-  missedStops?: ReactNode
+  missedStops?: Stop[]
```

Comment:
**praise:** I like this change. Makes it a lot easier to think about if `missedStops` is _just data_, rather than rendering logic as well.